extensions/arc/deployment/src/main/resources/dev-templates/beanDependencyGraph.html

```diff
@@ -137,22 +162,47 @@
         </div>
     </div>
     <div class="row">
-        <div class="col-md-auto">
+        <div class="col-2">
             <ul class="list-group list-group-flush">
                 <li class="list-group-item"><i class="fas fa-circle text-danger fa-sm legend-root"></i> root</li>
                 <li class="list-group-item"><i class="fa fa-long-arrow-right fa-sm legend-direct-dependency"></i> direct dependencies</li>
                 <li class="list-group-item"><i class="fa fa-long-arrow-right fa-sm legend-direct-dependent"></i> direct dependents</li>
                 <li class="list-group-item"><i class="fa fa-long-arrow-right fa-sm legend-dependency"></i> dependencies</li>
                 <li class="list-group-item"><i class="fa fa-long-arrow-right fa-sm legend-producer"></i> declaring bean of a producer</li>
                 <li class="list-group-item"><i class="fa fa-long-arrow-right fa-sm legend-lookup"></i> potential dependency <br/> &nbsp;<small class="text-muted">programmatic lookup</small></li>
+                <li class="list-group-item">
+                    <button class="btn btn-primary" type="button" id="zoom_in">
+                        <i class="fas fa-search-plus"></i> Zoom in
+                    </button>
+                    <button class="btn btn-primary" type="button" id="zoom_out">
+                        <i class="fas fa-search-minus"></i> Zoom out
+                    </button>
+                </li>
+                <li class="list-group-item">
+                <form method="post" action="toggleBeanDescription?beanId={bean.id}">
+                    <button class="btn btn-secondary" type="submit" title="Simple description does not include the package information">
+                        {#if info:devBeanInfos.beanDescription eq "simple"}<i class="fas fa-toggle-on"></i>{#else}<i class="fas fa-toggle-off"></i>{/if}
+                        Simple description {#if info:devBeanInfos.beanDescription eq "simple"}ON{#else}OFF{/if}
+                    </button>
+                </form>
```

Comment:
```suggestion
<form id="simpleDescriptionForm" method="post" action="toggleBeanDescription?beanId={bean.id}">
    <div class="form-group form-check">
        <input type="checkbox" class="form-check-input" id="simpleDescriptionCheck" title="Simple description does not include the package information" onclick="document.getElementById('simpleDescriptionForm').submit();">
        <label class="form-check-label" for="simpleDescriptionCheck">Simple description</label>
    </div>    
</form>
```

I am not sure if above will work (please test and see) but I think a checkbox makes more sense ? w.d.y.t ? We might need some more js to make this happen

Reply:
+1, a checkbox makes more sense.